javascript/javascript1/week3/jsweek3hw.js

```diff
@@ -0,0 +1,31 @@
+const names = [
+  "Peter",
+  "Ahmad",
+  "Yana",
+  "kristina",
+  "Rasmus",
+  "Samuel",
+  "katrine",
+  "Tala",
+];
+const nameToRemove = "Ahmad";
+
+// Write some code here
+function removeNameFromArray(name) {
+  if (arguments.length < 1 || typeof name != "string") {
+    console.log("Please provide the name to remove as a string");
+    return;
+  }
+  //the check is not necessary but just additional info for the user if required
+  if (names.indexOf(name) === -1) {
+    console.log(`${name} is not present in the array`);
+    return;
+  }
+  names.splice(names.indexOf(name), 1);
+}
+removeNameFromArray();
+removeNameFromArray("5");
+removeNameFromArray(nameToRemove);
+// Code done
+
+console.log(names); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
```

Comment:
Great work ✨
I would suggest an - if, else if, else - structure instead of your return statements on line 17 and 22 😊

Reply:
Hi Loui,
   thank you so much for reviewing the code. Is there any performance issue with return statements? If not isn't better with readability of the code than putting the major functionality within an else block? Just my view correct me if I am wrong. Thank you

Reply:
Hi Reenu, 
That's a great question. From my point of view, your code does not have performance issues. As you know a return statement can be used in many ways, also to interrupt a function, like you do in your example. 
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/return

For me, the readability in this instance would be better with an - if, else if, else - statement. But other developers might prefer your version 😊

Reply:
Thank you for your reply. I can rewrite it .